### DISCUSSION TITLE
Storing a JSON Array as a content item using Workflow

### DISCUSSION THREAD

**Discussion by MichaelNaghavipour** [OrchardCMS/OrchardCore#15870] [Q&A]
Hi everyone,

I am implementing an IoT platform using OrchardCore. At the current step, my client sends JSON data to my site, which I need to store as content items (which are named data & temp). The data is a single number, and the temp is an array of 24 numbers.

Here is my question:
How can I store **temp** with all its numbers into a content item with the same size and the **data** itself to another content?

Here is my JSON example which is being sent to my website by HTTP POST request:

**{
    "data": {
        "dataid":23,
        "temp": [1.0,0.2,0.0,0.4,0.0,4.0,1.3,0.0,0.8,7.2,0.5,1.0,4.3,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.1,0.1,5.0]
}
}**

I need to implement this using **Workflow**, and due to very limited OrchardCore documentation, it would be great if someone provides an example of storing such data using **Workflow** so that I could refer to it.

Any help would be greatly appreciated.

**Comment by sebastienros** [comment]
The CreateContentTask accepts json as input that can be built with these arguments.
Maybe using two tasks would create two content items.

**Comment by VahidFPA** [comment]
This will help you
![Receive Data From IoT](https://user-images.githubusercontent.com/3223934/89053274-3659b980-d36c-11ea-986e-01a5c3517cdf.jpg)
